apps/docs/src/routes/playground/docgen.svelte.ts

```diff
@@ -27,7 +27,7 @@ export class Docgen {
 			await tsvfs.createDefaultMapFromCDN(
 				this.#compiler_options,
 				ts.version,
-				false,
+				true,
```

Comment:
@ciscorn Before merging, one last question. Was there any good reason why you didn't have `cache` _(from `createDefaultMapFromCDN)_ enabled at start?

Reply:
Ah, sorry about that! I probably set it to `false` while testing whether `target: ESNext` was usable and forgot to switch it back. It should work perfectly fine with `true`.